lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -452,8 +452,9 @@ getLegacyTransaction tid = do
     let wits = mempty
     pure (Tx tid inps outs, wits)
 
-putSignedTx :: [(TxIn, Coin)] -> [TxOut] -> [TxWitness] -> Put
-putSignedTx inputs outputs witnesses = do
+putSignedTx :: ByteString -> [(TxIn, Coin)] -> [TxOut] -> [TxWitness] -> Put
+putSignedTx payload inputs outputs witnesses = do
+    putByteString payload
```

Comment:
The `payload` here is actually what `putStakeCertificate` inserts in `putStakeDelegationTx`. So we should either have two separate functions `putSignedTx` and `putStakeDelegationTx` with slightly different arguments, or a single function that takes an addition `Maybe payload` as argument. But not both :sweat_smile: